### PR TITLE
Dns views

### PR DESCRIPTION
**Pull request by eravin-ns1**
* adds support for DNS views - this is an updated version of #147 but now up to date with current v2 branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,87 @@ func main() {
...
+===========================
+
+DNS views is a means for NS1 to serve one set of data to one group of clients
+(e.g. internal employees), and different sets of data to other groups of
+clients (e.g. public internet). This has been largely exposed by allowing zones
+in the NS1 system to share the same FQDN, and allowing propagation to be
+controlled via ACLs and "views". For existing zones, and users that have no
+need for the added complexity of views, the default behavior is unchanged.
+However, it is important to understand that the requirement that an FQDN be
+unique within the network is removed in v3.x, and the ramifications of that.
```
Comment (Zach-Johnson) [inline README.md]: this sounds like a ddi specific comment, I think it should be removed

File: README.md [hunk 2]
```diff
@@ -53,6 +53,87 @@ func main() {
...
+
+Views are available in API v3.x. For more information, see:
```
Comment (Zach-Johnson) [inline README.md]: same

File: README.md [hunk 3]
```diff
@@ -53,6 +53,87 @@ func main() {
...
+https://help.ns1.com/hc/en-us/articles/360054071374
+
+COMPATIBILITY
```
Comment (Zach-Johnson) [inline README.md]: this seems like generally more information than needs to be in the SDK, this should be captured in API docs if it isn't. I don't think all of this belongs here. If you disagree, then all of the ddi-specific language needs to be changed.

**Review by Zach-Johnson — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (eravin-ns1) [reply README.md]: I think we need a short discussion of views and how view-aware code differs from older code, and the rest can be dropped.

Comment (eravin-ns1) [reply README.md]: README is updated, PTAL

File: rest/client.go [hunk 4]
```diff
@@ -12,7 +12,7 @@ import (
 )
 
 const (
-	clientVersion = "2.6.5"
+	clientVersion = "2.6.6"
```
Comment (Zach-Johnson) [inline rest/client.go]: changelog should be updated alongside a version bump. I would suggest keeping these in a separate PR rather than in a feature PR

Comment (eravin-ns1) [reply rest/client.go]: Done.

**Review by Zach-Johnson — APPROVED** [review]

**Pull request merged**